google-account-plugin/src/com/google/cloud/tools/intellij/login/GoogleLoginUtils.java

```diff
@@ -53,8 +53,7 @@
    * @param pictureCallback the user image will be set on this callback
    */
   @SuppressWarnings("FutureReturnValueIgnored")
-  public static void provideUserPicture(Userinfoplus userInfo,
-      final IUserPropertyCallback<Image> pictureCallback) {
+  public static void provideUserPicture(Userinfoplus userInfo, Consumer<Image> pictureCallback) {
```

Comment:
Yeah I agree, a lot of this would change if we decide to lean on the google-auth-library.